src/RageFileManager.h

```diff
@@ -27,6 +27,7 @@ class RageFileManager
 		vector<RString> const& ExtensionList, vector<RString> &AddTo,
 		bool bOnlyDirs= false, bool bReturnPathToo= false);
 	bool Move( const RString &sOldPath, const RString &sNewPath );
```

Comment:
Kind of separate, but related -- can we rename these to be `sFromPath` and `sToPath` to match `Copy()`

Reply:
Yeah, I can do that, even though I try to avoid that for new code.

Reply:
Changed!